bootstrap/src/main/java/io/jenkins/jenkinsfile/runner/bootstrap/Bootstrap.java

```diff
@@ -74,6 +74,10 @@
             "Note that the folder specified via --runHome will not be disposed after the run.")
     public File runHome;
 
+    @CheckForNull
+    @Option(name = "--withHooks", usage = "Path to a directory containing Groovy Hooks to copy into init.groovy.d")
```

Comment:
Rename to 'withInitHooks'? There are other types of hooks which exist in Jenkins. Boot Failure hooks would be reasonable for some JFR use-cases

Reply:
makes sense